core/src/main/java/org/apache/calcite/tools/RelBuilder.java

```diff
@@ -2602,6 +2611,32 @@ private RelBuilder aggregate_(GroupKeyImpl groupKey,
     return project(projects.transform((i, name) -> aliasMaybe(field(i), name)));
   }
 
+  public static Set<Integer> getAllFields2(ImmutableBitSet groupSet,
```

Comment:
change method name to something meaningful.

Reply:
resolved